agenta-web/src/components/pages/evaluations/NewEvaluation/SelectVariantSection.tsx

```diff
@@ -115,13 +135,14 @@ const SelectVariantSection = ({
 
     return (
         <Collapse
-            defaultActiveKey={["1"]}
-            {...props}
+            className={classes.collapseContainer}
+            activeKey={activePanel === "variantPanel" ? "variantPanel" : undefined}
+            onChange={() => handlePanelChange("variantPanel")}
             items={[
```

Comment:
Sorry if I wasn't clear on the first review. What I meant before was memoization of the items array that are passed to collapsed components. As we saw during the call, using that practice will also bring performance updates during collapse ui state changes. 🙏